MoneyLoverClone/Scenes/ChangePassWord/ChangePassViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  ChangePassViewController.swift
+//  MoneyLoverClone
+//
+//  Created by nguyen viet hoang on 9/29/20.
+//  Copyright © 2020 Vu Xuan Cuong. All rights reserved.
+//
+
+import UIKit
+import Reusable
+
+class ChangePassViewController: UIViewController {
+
+    @IBOutlet private weak var tfOldPass: UITextField!
+    @IBOutlet private weak var tfNewPass: UITextField!
+    @IBOutlet private weak var tfConfirmPass: UITextField!
+    @IBOutlet private weak var btnSave: UIButton!
+    var placeholderLabel : UILabel!
+    var currentPass: String = ""
+    var textPass = ""
+    var textNewPass = ""
+    var textConfirm = ""
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setTextPlaceHolder()
+        print(currentPass)
+    }
+    
+    func setTextPlaceHolder() {
+        setTextField(tfOldPass , input: "Nhập mật khẩu")
```

Comment:
```suggestion
        setTextField(tfOldPass, input: "Nhập mật khẩu")
```
tương tự các dòng dưới